libbeat/processors/actions/docs/append.asciidoc

```diff
@@ -0,0 +1,54 @@
+[[append]]
+=== Append Processor
+
+++++
+<titleabbrev>append_processor</titleabbrev>
+++++
+
+The `append_processor` processor appends static values and values from the listed fields to target field.
```

Comment:
Can you expand description so it is like the `append` ingest node processor?  I think we want to behavior of these two `append` implementations to be the same.

https://www.elastic.co/guide/en/elasticsearch/reference/current/append-processor.html

Reply:
Actually, I have that particular processor in mind while developing this one. I'll update the description in this one though. 

PS. could you please let me know if I'm missing any other doc file? CHANGELOG or any other? I'm unaware of the doc files that needed to be changed for libbeat. 😅